nginx.rb

```diff
@@ -17,6 +17,7 @@
 set_default(:domain_name, "#{nginx_config_name}")
 set_default(:nginx_access_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_access.log")
 set_default(:nginx_error_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_error.log")
+set_default(:nginx_proxy_read_timeout, 75)
```

Comment:
OK, How about this?
The default value is set here like `set_default(:nginx_proxy_read_timeout, 30)`.
And the specific value for rails is set in here config/rails.rb https://github.com/Coiney/capiche-coiney-config/blob/master/rails.rb